### ISSUE TITLE
Misalignment of the pulmonary veins

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6229]
**Preferred term label:**
Misalignment of the pulmonary veins

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect of pulmonary veins characterized by abnormal location of pulmonary vein branches adjacent to pulmonary arteries within the same adventitial sheath.

Comment -- describe the normal anatomy of the pulmonary veins ?

@crapp, @mgriese, @katrinknoflach please revise definition as necessary

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0011718

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/265380

7/14, PMID: 19500772

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033186 for upcoming release, revise definition as needed.

**Comment by mgriese** [comment]
Definition (free text, please give PubMed ID)
The term is commonly used to describe a putative abnormal location of pulmonary vein branches adjacent to pulmonary arteries within the same adventitial sheath (PMID: 27071622). Evidence has been provided that the vessels in question are not pulmonary veins, however represent dilated bronchial veins (PMID: 25052575). 

Disease association; PMID: 27071622; PMID: 23505205;

**Comment by pnrobinson** [comment]
updated definition, thanks!
